core/controlplane/config/templates/cloud-config-controller

```diff
@@ -594,17 +605,15 @@ write_files:
          echo 'Failed to create the cluster role named "cluster-admin". It is ok if you are using k8s newer than 1.5'
       fi
 
-      for manifest in {bootstrapped-node,}; do
-          kubectl apply -f "${mfdir}/cluster-roles/$manifest.yaml"
-      done
+      kubectl apply -f "${mfdir}/cluster-roles/node-extensions.yaml"
```

Comment:
will it be beneficial to install RBAC roles before installing kube-dns and kube-proxy?